extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosExporter.java

```diff
@@ -387,22 +408,27 @@ private List<PhotoModel> convertMediaItemsToPhotoModels(
   private static PhotoModel convertToPhotoModel(
       GoogleMediaItem mediaItem,
       Optional<String> albumId,
-      TempMediaData tempMediaData) throws InvalidExportedItemException {
+      TempMediaData tempMediaData) throws InvalidExportedItemException, ParseException {
     boolean shouldUpload = albumId.isPresent();
     if (tempMediaData != null) {
       shouldUpload = shouldUpload || !tempMediaData.isContainedPhotoId(mediaItem.getId());
     }
 
     if (!shouldUpload)
       throw new InvalidExportedItemException(
-              String.format("Failed to convert media item (id: %s) into a PhotoModel", mediaItem.getId())
+          String.format("Failed to convert media item (id: %s) into a PhotoModel", mediaItem.getId())
       );
 
     try {
       return GoogleMediaItem.convertToPhotoModel(albumId, mediaItem);
     } catch (IllegalArgumentException e) {
       throw new InvalidExportedItemException(
-              String.format("Failed to convert media item (id: %s) into a PhotoModel", mediaItem.getId())
+          String.format("Failed to convert media item (id: %s) into a PhotoModel", mediaItem.getId())
+      );
+    } catch (ParseException e) {
+      throw new ParseException(
+          String.format("Parse exception occurred while converting MediaItem (id: %s), skipping this item. "
+          + "Failure message : %s", mediaItem.getId(), e.getMessage()), e.getErrorOffset()
```

Comment:
lets make these into 1 catch block, I think its fine to just log an InvalidExportedItemException here with the parse exception as the cause

Reply:
Done